ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml

```diff
@@ -248,13 +248,13 @@ StatusModal {
                                 type: StatusMenuItem.Type.Danger
                                 //% "Remove From Group"
                                 text: qsTrId("remove-from-group")
-                                onTriggered: popup.chatSectionModule.removeMemberFromGroupChat(popup.chatDetails.id,  model.id)
+                                onTriggered: popup.chatSectionModule.removeMemberFromGroupChat("", popup.chatDetails.id,  model.id)
```

Comment:
I think both those functions should be changed so that the QML stays like it was before. It should be the controller's job to add the community ID.